app/views/works/_meta_tags.html.erb

```diff
@@ -31,13 +31,6 @@ fb:app_id Facebook ID A Facebook app ID
 fb:pages  Array<Facebook ID>
 fb:profile_id %>
 
-<meta name="twitter:card" content="summary_large_image">
-<meta name="twitter:site" content="@<%= Rails.application.config.twitter_acct %>"/>
-
-<%# not sure twitter really uses these, but maybe it used to? %>
-<meta name="twitter:label1" content="Rights Statement"/>
-<meta name="twitter:data1" content="<%= attributes.rights_statement %>"/>
-
```

Comment:
I think we should leave these -- We don't have a twitter share button anymore, but someone, anyone, without our coordination, can still share our page to twitter, and when they do, we should still provide the meta tags to enable twitter to produce a good tweet.